### PR TITLE
Release 3.1.0

### PR DESCRIPTION
**Pull request by cabas**
* Added global configuration for @testing-library/jest-dom
* Added ignore patterns
* Added url regex
* Added ColorPickerInput component
* Added FormElement component for react-hook-forms
* Changed some components proptypes
* Changed react-testing-library for @testing-library/react

### CODE REVIEW

File: javascript/components/inputs/tests/ColorPickerInput.test.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  describe('when is uncontrolled', () => {
+    beforeEach(() => {
+      ref = undefined;
+      wrapper = render(<ColorPickerInput {...props} />);
+    });
+    afterAll(cleanup);
+
+    it('renders correctly', () => {
+      expect(wrapper).toMatchSnapshot();
+    });
```
Comment (ccanalesb) [inline javascript/components/inputs/tests/ColorPickerInput.test.js]: Esto es necesario?

File: javascript/components/inputs/ColorPickerInput.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import style from './style.css';
+
+const ColorPickerInput = ({ className, onClose, onChange, inputRef, name, controlled }) => {
+  const [displayColorPicker, setDisplayColorPicker] = useState(false);
+  const [backgroundColor, setBackgroundColor] = useState('');
+  const handleColorChange = ({ hex }) => {
+    setBackgroundColor(hex);
+    onChange(hex);
+  };
+  return (
```
Comment (raicp) [inline javascript/components/inputs/ColorPickerInput.js]: me cuesta un poco entender cuando están sin `new line` antes. Quizás es constumbre

File: javascript/components/inputs/tests/ColorPickerInput.test.js (hunk 1 above)
Comment (cabas) [reply javascript/components/inputs/tests/ColorPickerInput.test.js]: probably not, lo sacaré

File: javascript/components/inputs/ColorPickerInput.js (hunk 2 above)
Comment (cabas) [reply javascript/components/inputs/ColorPickerInput.js]: que quisquilloso :|

**Pull request merged**